src/core/abr/network_analyzer.ts

```diff
@@ -67,7 +67,7 @@ function getConcernedRequests(
     }
     const segmentEnd = request.time + request.duration;
     return segmentEnd > neededPosition &&
-           Math.abs(neededPosition - request.time) < -0.3;
+           Math.abs(neededPosition - request.time) > -1.2;
```

Comment:
What ? `Math.abs(x) >= 0 === true`, no ?

Reply:
Ha you're right!
I didn't realize that the `Math.abs` was here! The old like the new code makes no sense here

Reply:
Done